src/assets/stylesheets/layouts/layout-auth.scss

```diff
@@ -0,0 +1,50 @@
+.layout-auth {
+  &,
+  body,
+  .root-container {
+    height: 100%;
+  }
+
+  .root-container {
+    display: flex;
+  }
+
+  body {
+    background: {
+      image: url('/assets/images/background.png');
+      size: cover;
+
+      position: center;
+    }
+  }
+
+  .app-header-title {
+    padding-top: rem(24px);
+
+    opacity: 0.6;
+
+    color: #fff;
+    font-family: $font-family-sans-serif-normal;
+  }
+
+  .app-content {
+    z-index: 1;
+    width: rem(330px);
+    flex-direction: column;
+    align-items: flex-start;
+    padding-top: rem(145px);
+    margin: 0 auto;
+    background-color: inherit;
+    text-align: center;
+
+    // @media screen and (max-width: 768px) {
```

Comment:
Why did you comment these code?
But for responsive, our team prefer mobile-first, so the default is for mobile screen, and to support the larger screens, you can use some breakpoint mixins that was created by Bootstrap to use: https://getbootstrap.com/docs/5.2/layout/breakpoints/#min-width

Reply:
I did not mean to push these code sorry

Reply:
I've made some changes and removed the comment in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/6f7d8911e5efec5fc7ac24ea69f8d8a49e9dda7d#diff-7bda7f21f2c7a2bcf364bff7ef5618cd2aeaa46989b5f8d00399404a52e327ba